lms/djangoapps/discussion/tests/test_tasks.py

```diff
@@ -19,7 +19,7 @@
 import openedx.core.djangoapps.django_comment_common.comment_client as cc
 from common.djangoapps.student.tests.factories import CourseEnrollmentFactory, UserFactory
 from lms.djangoapps.discussion.signals.handlers import ENABLE_FORUM_NOTIFICATIONS_FOR_SITE_KEY
-from lms.djangoapps.discussion.tasks import _should_send_message, _track_notification_sent
+from lms.djangoapps.discussion.tasks import _is_first_comment, _should_send_message, _track_notification_sent
```

Comment:
consider breaking into new lines
```suggestion
from lms.djangoapps.discussion.tasks import (
  _is_first_comment,
  _should_send_message,
  _track_notification_sent,
)
```

Reply:
Changed.